build.sbt

```diff
@@ -3,7 +3,7 @@ val metricsVersion = "3.1.0"
 val baseScalaVersion = "2.11.8"
 
 lazy val publishSettings = Seq(
-  version := "0.3.0-M3",
+  version := "0.3.0-M4",
```

Comment:
don't change the version here. that should be done in a dedicated release PR. Perhaps we should follow java lib SNAPSHOT version conventions.